test/integration_test/basic_test.go

```diff
@@ -415,19 +415,29 @@ func BasicUpgradeOperator(tc *types.TestCase) func(*testing.T) {
 
 		var lastHopImage string
 		for _, hopImage := range ci_utils.OperatorUpgradeHopsImageList {
-			pxOperatorDeployment, err := ci_utils.GetPxOperatorDeployment()
+			pxOperatorDep := &appsv1.Deployment{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "portworx-operator",
+					Namespace: cluster.Namespace,
+				},
+			}
+			pxOperatorDeployment, err := ci_utils.GetPxOperatorDeployment(pxOperatorDep)
 			require.NoError(t, err)
-			pxOperatorImage, err := ci_utils.GetPxOperatorImage()
+			pxOperatorImage, err := ci_utils.GetPxOperatorImage(pxOperatorDeployment)
 			require.NoError(t, err)
+			if len(lastHopImage) == 0 {
+				// This is initially deployed PX Operator version
+				lastHopImage = pxOperatorImage
+			}
 
 			if pxOperatorImage == hopImage {
-				logrus.Infof("Skipping upgrade of PX Operator from %s to %s", pxOperatorImage, hopImage)
+				logrus.Infof("Skipping upgrade of PX Operator from [%s] to [%s]", pxOperatorImage, hopImage)
```

Comment:
Just a suggestion - You can use `%q` to get a quoted string in the output if you want.
For example,
```
logrus.Infof("Skipping upgrade of PX Operator from %q to %q", pxOperatorImage, hopImage)
```
Output will be:
```
INFO Skipping upgrade of PX Operator from '23.4.0' to '23.5.0'
```